packages/eslint-config-react-a11y/test/fixtures/rules/jsx-a11y/jsx-a11y.js

```diff
@@ -0,0 +1,348 @@
+import React from 'react';
+
+// '/accessible-emoji': Enforce emojis are wrapped in and provide screen reader access.
```

Comment:
Can we keep these "separators" consistent with the other packages? see: https://github.com/moxystudio/eslint-config/blob/master/packages/eslint-config-react/test/fixtures/rules/react/react.js#L5